client/performance-profiler/pages/weekly-report/unsubscribe.tsx

```diff
@@ -0,0 +1,81 @@
+import { useTranslate } from 'i18n-calypso';
+import { useEffect } from 'react';
+import DocumentHead from 'calypso/components/data/document-head';
+import { useUnsubscribeMutation } from 'calypso/data/site-profiler/use-unsubscribe-query';
+import {
+	MessageDisplay,
+	ErrorSecondLine,
+} from 'calypso/performance-profiler/components/message-display';
+import LoaderText from 'calypso/performance-profiler/components/weekly-report/loader-text';
+import { WeeklyReportProps } from 'calypso/performance-profiler/types/weekly-report';
+
+export const WeeklyReportUnsubscribe = ( props: WeeklyReportProps ) => {
+	const translate = useTranslate();
+	const { url, hash } = props;
+
+	const siteUrl = new URL( url );
+
+	const { mutate, isPending, isError, isSuccess } = useUnsubscribeMutation( url, hash );
+
+	useEffect( () => {
+		mutate();
+	}, [ mutate ] );
+
+	const secondaryMessage = translate(
+		'You can always resubscribe to receive performance change emails by opting in again for weekly reports at any time.'
```

Comment:
Nit: I think the terms "always" and "at any time" are suggesting the same, and this sentence feels a bit overcomplicated to me. I'd use something like:
```suggestion
		'You can opt in again for weekly reports to receive performance change emails.'
```